qadence/extensions.py

```diff
@@ -67,6 +77,7 @@ def _set_backend_config(backend: Backend, diff_mode: DiffMode) -> None:
     """
 
     _validate_diff_mode(backend, diff_mode)
+    _validate_backend_config(backend)
```

Comment:
@madagra @nmheim @Roland-djee lets handle the deprecation of grad_ckpting like this

Reply:
its convert_state now